examples/frequencyseries/inject.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# Copyright (C) Alexander Urban (2018-2019)
+# Copyright (C) Alexander Urban (2018-2020)
```

Comment:
Can we change this for consistency with the others (and to match my preference)?

```suggestion
# Copyright (C) Alex Urban (2018-2020)
```

Reply:
I have applied this suggestion, but please note that it is extremely unlikely that you are actually eligible to own the copyright of this file, in the same way that I am probably not eligible to own the copyright on the rest of the codebase. These rights likely go to Cardiff and LSU as appropriate, but they are unlikely to ever care.